frontend/src/app/components/wp-card-view/wp-card-view.component.sass

```diff
@@ -89,3 +89,9 @@ wp-edit-field
 
 .wp-inline-create--reference-container
   margin-bottom: 3rem
+
+.wp-card--cover-image
+  display: block
+  margin: -10px -10px 10px
+  width: calc(100% + 10px + 10px)
+  max-width: calc(100% + 10px + 10px)
```

Comment:
Is there a reason you let the image span the complete width without any space to the card border? These calculations are quite hacky and will cause problems sooner or later. The closing icon is, for example, now shown on the image. I would suggest to change it to `margin: 0 auto 10px auto` and remove the width rules completely.

Reply:
@HDinger Thanks for looking into the is PR. The reason is that it simply looks better to not have any padding around the image.

The icon for removing a card will need some background indeed.

Yes, it is a bit hacky. To improve it I would probably need to encapsulate the lower part and give that a padding and place the cover image above, without a padding.

Reply:
![image](https://user-images.githubusercontent.com/327272/59750347-f4aca300-927e-11e9-92c4-4f516f98de84.png)